### PR TITLE
dts: alcatel-idol3: Use panel selection

### PR DESCRIPTION
**Pull request by vknecht**

### CODE REVIEW

**Review by TravMurav — APPROVED** [review]

File: dts/msm8916/msm8939-mtp.dts [hunk 1]
```diff
@@ -12,7 +12,24 @@
...
+		lk2nd,match-panel;
+
+		panel {
+			compatible = "alcatel,idol3-panel";
+
+			qcom,mdss_dsi_jdi_1080p_video {
+				compatible = "alcatel,idol3-panel-jdi";
+			};
+			qcom,mdss_dsi_nt35596_1080p_video {
+				compatible = "alcatel,idol3-panel-nt35596";
```
Comment (Minecrell) [inline dts/msm8916/msm8939-mtp.dts:24]: Do you have this panel? Not sure if we should support it, perhaps this is just some early prototype?

Comment (vknecht) [reply dts/msm8916/msm8939-mtp.dts:24]: My devices have nt35596 (not v03) and r63315, and saw no evidence of others being used.
Only those two have generated drivers... Should the others be removed ?

Comment (Minecrell) [reply dts/msm8916/msm8939-mtp.dts:24]: The jdi one looks like an exact copy from the qcom tree, it probably doesn't actually exist. I'd say we can remove that one. Not sure about the v3, you can keep it or remove it. Toss a coin maybe :D

Comment (vknecht) [reply dts/msm8916/msm8939-mtp.dts:21]: Ok I removed the jdi one

**Review by Minecrell — APPROVED** [review]

**Pull request merged**